### PR TITLE
Adding more tests in testlogger & renaming receivers.

### PR DESCRIPTION
**Pull request by miccagiann**
I have renamed the receivers in `testlogger.zzzgo` file. I have added more tests to cover the missing methods for `testlogger`. I have completed 100% coverage for `support.go` file... Still I am wrapping my head on how to test only the methods for `seed.go`, `span.go` and `config.go`.

### CODE REVIEW

File: support_test.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ func TestBase26(t *testing.T) {
 		26: "BA",
 		27: "BB",
 		52: "CA",
+		-1: "-B",
```
Comment (muir) [inline support_test.go:17]: Hmm.   Is that the right answer?
The numbers as letters are funny.  I want to start counting with "A" for 0.  Maybe "-A" is a better answer for -1 though.

File: xoptest/testlogger_test.go [hunk 2]
```diff
@@ -1,42 +1,85 @@
...
 	}
+	tLog := xoptest.New(t)
+	defer tLog.Close()
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		err := tLog.WithLock(f)
+		assert.NoError(t, err)
+	}()
+	wg.Add(1)
```
Comment (muir) [inline xoptest/testlogger_test.go:64]: If you `wg.Add()` after the count gets down to zero, `Add` will panic.  Use `wg.Add(2)` up above and drop this one to eliminate the possible race condition.

**Review by muir — APPROVED** [review]
This needs at least one little fix before merging.  The numbering for -1 isn't important enough to worry about.

File: support_test.go (hunk 1 above)
Comment (miccagiann) [reply support_test.go:17]: Okay, marking this as resolved (won't fix in this PR).

**Pull request merged**